sdcm/utils/udf.py

```diff
@@ -40,11 +67,12 @@ def from_yaml(cls, udf_yaml_filename: str):
 
 
 def _load_all_udfs():
+    """Convenience functions for loading all the existing UDF scripts from /sdcm/utils/udf_scripts"""
```

Comment:
Add also type hints, so using `UDFS` will be easier.

Reply:
Fixed (also added to UDF methods)